app/assets/javascripts/discourse/app/lib/user-presence.js

```diff
@@ -1,68 +1,134 @@
-// for android we test webkit
-const hiddenProperty =
-  document.hidden !== undefined
-    ? "hidden"
-    : document.webkitHidden !== undefined
-    ? "webkitHidden"
-    : undefined;
+import { isTesting } from "discourse-common/config/environment";
 
-const MAX_UNSEEN_TIME = 60000;
+const callbacks = [];
```

Comment:
Should we need to clear this between tests? Usually it's a good idea with module local variables like this.

Reply:
Updated in latest commit